### PR TITLE
[DEVOPS-393] Manually trigger build after export-config

### PR DESCRIPTION
**Pull request by FleetAdmiralButter**
1) Manually call the `build` workflow after running the export-config.sh script, as commits published as part of an Actions run won't trigger workflows that would ordinarily run with `on: push`.

### CODE REVIEW

File: .github/workflows/export_config.yml [hunk 1]
```diff
@@ -38,4 +38,14 @@ jobs:
...
+      - name: Get the release branch
+        run: |
+          CURRENT_GIT_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+          echo RELEASE_GIT_BRANCH="${CURRENT_GIT_BRANCH//automation/release}" >> $GITHUB_ENV
+      - name: Manually run build workflow
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: build
+          repo: ${{ secrets.GITHUB_REPOSITORY }}
+          ref: ${{ env.RELEASE_GIT_BRANCH }}
```
Comment (FleetAdmiralButter) [inline .github/workflows/export_config.yml:51]: @GROwen I did some more testing and it turns out the workflow_dispatch does pick up the correct commit after all when you pass in the release branch via the `ref` parameter.

File: .github/workflows/export_config.yml [hunk 2]
```diff
@@ -38,4 +38,14 @@ jobs:
...
+          /app/.circleci/export-config.sh
+      - name: Get the release branch
+        run: |
+          CURRENT_GIT_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+          echo RELEASE_GIT_BRANCH="${CURRENT_GIT_BRANCH//automation/release}" >> $GITHUB_ENV
+      - name: Manually run build workflow
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: build
+          repo: ${{ secrets.GITHUB_REPOSITORY }}
```
Comment (nicksantamaria) [inline .github/workflows/export_config.yml:50]: Why is `GITHUB_REPOSITORY` in`secrets` ?

File: .github/workflows/set_status.yml [hunk 3]
```diff
@@ -36,7 +36,8 @@ jobs:
       - name: Set commit status in the GitHub API
         run: |
           curl -X POST \
-          -H "Authorization: token ${{ secrets.SDPDEPLOY_PAT }}" \
+          -vvv \
+          -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
```
Comment (GROwen) [inline .github/workflows/set_status.yml:40]: :heart:

File: .github/workflows/export_config.yml (hunk 2 above)
Comment (FleetAdmiralButter) [reply .github/workflows/export_config.yml]: Typo fixed!

**Review by nicksantamaria — APPROVED** [review]

**Review by steveworley — APPROVED** [review]

**Pull request merged**